docs/api/en/objects/BatchedMesh.html

```diff
@@ -129,6 +129,19 @@ <h3>[method:this setCustomSort]( [param:Function sortFunction] )</h3>
 			in the list include a "z" field to perform a depth-ordered sort with.
 		</p>
 
+		<h3>
+			[method:undefined getColorAt]( [param:Integer index], [param:Color color] )
+		</h3>
+		<p>
+			[page:Integer index]: The index of an instance. Values have to be in the
+			range [0, count].
```

Comment:
We should update the copy in this documentation to not use `instancing` nomenclature.

Reply:
I could change `instancing` nomenclature in favor of `geometry` nomencalture, the only problem I see is the class has two methods called `getInstanceCountAt` and `setInstanceCountAt`, which also appear in the documentation. So to be consistent we should also rename both methods, but that would introduce a breaking change.
Do you wish me to change that or do we keep it as it is for now?

Reply:
Those functions correctly refer to "instances" and do not need to change.

Reply:
I may be misunderstanding then. What copy do you want me to change and which one not?

Reply:
Sorry if I was unclear - we only need to change the words "instances" that were added in this PR. Those other function names are correct.

Reply:
I think I understood now, done :)